modules/module-mongodb-storage/src/storage/implementation/MongoSyncBucketStorage.ts

```diff
@@ -315,8 +315,13 @@ export class MongoSyncBucketStorage
       });
     }
 
-    const limit = options?.limit ?? storage.DEFAULT_DOCUMENT_BATCH_LIMIT;
-    const sizeLimit = options?.chunkLimitBytes ?? storage.DEFAULT_DOCUMENT_CHUNK_LIMIT_BYTES;
+    // Internal naming:
```

Comment:
I wonder if it's worth renaming `storage.SyncBucketDataBatch` to `storage.SyncBucketDataChunk` to make this naming consistent everywhere. At least in my understanding, these chunks make up the batch, so yielding multiple things named "batch" from this method sounds like a deviation from this naming scheme.